test_client/src/main/scala/io/reactivesocket/tck/ClientDSL.scala

```diff
@@ -4,7 +4,7 @@ import java.io.{File, PrintWriter}
 
 class ClientDSL {
 
-  val writer: PrintWriter = new PrintWriter(new File(this.getClass.getSimpleName + ".txt"))
+  var writer: PrintWriter = new PrintWriter(new File(this.getClass.getSimpleName + ".txt"))
```

Comment:
I was messing around and trying to get something working, it is pretty hacky I agree, but my thought was that no one would be touching this class or the class that extends it and no one needs to set it, so it wasn't a big problem in my mind, but I will see if I can keep things elegant and make things less hacky.